### PR TITLE
Feature - GRPC Auth middleware

### PR DESCRIPTION
**Pull request by dannypaz**
This PR adds authorization middleware to each grpc call to allow users to authenticate a route.

Changes also include:
1. CHANGE_LOG
2. CONTRIBUTIONS file
3. Updated docs

### CODE REVIEW

**Comment by dannypaz** [comment]
[Add Authentication middleware to grpc-methods](https://trello.com/c/whr5USp4/664-add-authentication-middleware-to-grpc-methods)

File: CHANGE_LOG [hunk 1]
```diff
@@ -0,0 +1,6 @@
+v0.6.2 - dannypaz
+
+- Added authentication to grpc unary calls to allow the user to check metadata credentials BEFORE a route is submitted. This
```
Comment (martineehrlich) [inline CHANGE_LOG]: Only unary? Looks like it was also added to streaming?

Comment (martineehrlich) [inline CHANGE_LOG]: Only unary? Looks like it was also added to streaming?

**Comment by dannypaz** [comment]
[Add Basic auth between KCLI and KBD](https://trello.com/c/kcxsYPLg/13-add-basic-auth-between-kcli-and-kbd)

**Review by martineehrlich — APPROVED** [review]
LGTM

**Pull request merged**